pennylane/templates/embeddings/angle.py

```diff
@@ -136,3 +152,27 @@ def compute_decomposition(features, wires, rotation):  # pylint: disable=argumen
         features = math.T(features) if batched else features
 
         return [rotation(features[i], wires=wires[i]) for i in range(len(wires))]
+
+
+def _angle_embedding_resources(rotation: Operation, num_wires: int) -> dict:
+    return {resource_rep(rotation): num_wires}
+
+
+@register_resources(_angle_embedding_resources)
+def _angle_embedding_decomposition(features: list, wires: WiresLike, rotation: Operation):
```

Comment:
Is this generally correct?
```suggestion
def _angle_embedding_decomposition(features: list[float], wires: WiresLike, rotation: Operation):
```

Reply:
Sometimes its `list[float]`, sometimes `list[int]` sometimes `list[list[float], list[float]]` sometimes `list[list[int], list[int]]`, etc.

Reply:
All good then.